### ISSUE TITLE
vscodium adds microsoft apt repository 

### ISSUE THREAD

**Issue by cuechan** [VSCodium/vscodium#23]
When installing vscodium the microsoft apt repository is added to the system. I think this should not be done since most people are using vscodium because they don't want to use vscode from microsoft. 

https://github.com/Microsoft/vscode/blob/5db58b3867bfff05e6f4d874753ef960e9a59c4a/resources/linux/debian/postinst.template#L68

This issue is maybe related to #1 and #5

**Comment by stripedpajamas** [comment]
Good catch @cuechan 

It looks like it wouldn't fire that logic if the application name was 'code-oss', but we changed the application name to 'vscodium' pre build. I just opened PR #24 to resolve this. I also checked through the rest of the vscode repo to see if anything else was dependent on the name being (or not being) code-oss. The only other thing I found was [this commented out line](https://github.com/Microsoft/vscode/blob/5db58b3867bfff05e6f4d874753ef960e9a59c4a/resources/linux/rpm/code.spec.template#L37). If they do ever try to register the yum repository, we can update our side as well.

After the PR gets merged, I will delete the existing `.deb` file from VSCodium releases, so that the new clean version gets built. 

Thanks for pointing this out 👍
